src/main/hook/patch/gfx.c

```diff
@@ -30,11 +34,14 @@ typedef Window (*XCreateWindow_t)(
 typedef Display *(*XOpenDisplay_t)(const char *display_name);
 typedef XVisualInfo *(*glXChooseVisual_t)(
     Display *dpy, int screen, int *attribList);
+typedef void (*glXSwapBuffers_t)(Display *dpy, GLXDrawable drawable);
 
 static bool patch_gfx_initialized;
 static XCreateWindow_t patch_gfx_real_XCreateWindow;
 static XOpenDisplay_t patch_gfx_real_XOpenDisplay;
 static glXChooseVisual_t patch_gfx_real_GlXChooseVisual;
+static glXSwapBuffers_t patch_gfx_real_glXSwapBuffers;
+static uint16_t s_frame_limit = 0;
```

Comment:
Odd naming, why `s_`?

Reply:
Actually, I just realized the style here is `patch_gfx_xxx`. I suggest to stick to that to stay consistent for now.

```suggestion
static uint16_t patch_gfx_frame_limit = 0;
```

Reply:
s_ for a static internal to the file and not a global, but I'll update it

Reply:
however - that name also conflicts with the method to set the option on this file now, so it needs to be adjusted

Reply:
As discussed in chat, let's got for a cleaner direction of using a "option struct" to be passed into the init call. Idea from bemanitools which implements this in its even larger gfx hook backends.